derive/src/rpc_attr.rs

```diff
@@ -81,12 +84,18 @@ impl RpcMethodAttribute {
 								let aliases = get_meta_list(&meta).map_or(Vec::new(), |ml| get_aliases(ml));
 								let raw_params =
 									get_meta_list(meta).map_or(false, |ml| has_meta_word(RAW_PARAMS_META_WORD, ml));
+								let params_style = match raw_params {
+									true => Ok(Some(ParamStyle::Raw)),
```

Comment:
Can we print a deprecation warning here?

Reply:
Unfortunately I can't figure out a way to emit a warning without using the `proc_macro::Diagnostic` feature which is unstable

Not sure if we have any other ideas for this or if it is ok to just add a depreciation note to the docs but not the macro

Reply:
I think in the past we simply used to `println` the warning. @ascjones do you recall?